client/cmd/root.go

```diff
@@ -96,6 +103,8 @@ func init() {
 	rootCmd.AddCommand(sshCmd)
 	serviceCmd.AddCommand(runCmd, startCmd, stopCmd, restartCmd) // service control commands are subcommands of service
 	serviceCmd.AddCommand(installCmd, uninstallCmd)              // service installer commands are subcommands of service
+	upCmd.PersistentFlags().StringSliceVar(&natExternalIPs, externalIPMapFlag, nil, "Sets external IPs maps between local addresses and interfaces. You can specify a comma-separated list with a single IP and IP/IP or IP/Interface Name. e.g. --external-ip-map 12.34.56.78/10.0.0.1 or --external-ip-map 12.34.56.200,12.34.56.78/10.0.0.1,12.34.56.80/eth1")
+	upCmd.PersistentFlags().StringVar(&customDNSAddress, dnsResolverAddress, "", "Sets a custom address for NetBird's local DNS resolver. If set, the agent won't attempt to discover the best ip and port to listen on. e.g. --dns-resolver-address 127.0.0.1:5053")
```

Comment:
Would be great to mention that if an empty string is provided, it will clean the IPs, e.g.:
```
--external-ip-map ""
```